app/helpers/pagination_helper.rb

```diff
@@ -12,4 +12,14 @@ def per_page_options(options = [5, 10, 15, 25, 50])
     options << params[:per_page].to_i if params[:per_page].present?
     options.uniq.sort
   end
+
+  def react_pagination_props(collection = nil, classname = nil)
```

Comment:
I assume this will be not needed once we move to API based audits? (e.g. we would generate the pagination client side only?)

Reply:
yes, this can be removed in the future PR